backend/tilavarauspalvelu/admin/payment_order/admin.py

```diff
@@ -129,6 +114,25 @@ class PaymentOrderAdmin(admin.ModelAdmin):
 
     # Form
     form = PaymentOrderForm
+    fields = [
+        "reservation",
+        "remote_id",
+        "payment_id",
+        "refund_id",
+        "payment_type",
+        "status",
+        "price_net",
+        "price_vat",
+        "price_total",
+        "processed_at",
+        "language",
+        "reservation_user_uuid",
+        "checkout_url",
+        "receipt_url",
+    ]
+    readonly_fields = [
+        "reservation",
+    ]
```

Comment:
Now that `reservation` is read only, the `PaymentOrderForm.__init__` needs to be fixed, because `reservation` is no longer present in `self.fields` so trying to update it's `help_test` causes an error